### PR TITLE
SC Overview

### PR DESCRIPTION
**Pull request by kimsauce**
### Description

see https://saucedev.atlassian.net/browse/TCD-2383

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/secure-connections.md [hunk 1]
```diff
@@ -24,14 +24,14 @@ See our [Sauce Connect Proxy Security Overview](https://saucelabs.com/resources/
   <div className="box-wrapper" markdown="1">
     <div className="box box1 card">
       <div className="container">
-      <h3><a href="/secure-connections/sauce-connect">Using Sauce Connect Proxy</a></h3>
-      <p>Review the Sauce Connect Proxy system and network requirements, and then install the Sauce Connect Proxy client.</p>
+      <h3><a href="/secure-connections/sauce-connect">Sauce Connect Proxy Overview</a></h3>
+      <p>Learn about Sauce Connect Proxy system requirements and how to install the client.</p>
```
Comment (lysannep) [inline docs/secure-connections.md:28]:
```suggestion
      <p>See the Sauce Connect Proxy system requirements and learn how to install the client.</p>
```

Comment (lysannep) [inline docs/secure-connections.md:28]:
```suggestion
      <p>See the Sauce Connect Proxy system requirements and learn how to install the client.</p>
```
^The description for the overview doesn't seem to align with the content in the linked topic.

File: docs/secure-connections.md [hunk 2]
```diff
@@ -24,14 +24,14 @@ See our [Sauce Connect Proxy Security Overview](https://saucelabs.com/resources/
...
       </div>
     </div>
     <div className="box box2 card">
       <div className="container">
       <h3><a href="/secure-connections/sauce-connect/#setup-and-configuration">Setup and Configuration</a></h3>
-      <p>Find the right Sauce Connect Proxy tunnel configuration for you and your organization. Once you’re connected, use our code samples to run your first test.</p>
+      <p>Find the right Sauce Connect Proxy tunnel configuration for you and your organization.</p>
```
Comment (lysannep) [inline docs/secure-connections.md:34]: ^The description for the overview doesn't seem to align with the content in the linked topic -- the setup and configuration topic doesn't explicitly provide the info needed to determine which tunnel config to use (I found the info in this section: /secure-connections/#choosing-the-right-solution)

File: docs/secure-connections.md [hunk 3]
```diff
@@ -94,10 +95,11 @@ Sauce IPSec Proxy establishes a VPN connection between two IPSec gateways, and a
   </div>
 </div>
 
+For more information, see our [Sauce IPSec Proxy white paper](https://saucelabs.com/resources/white-papers/sauce-ipsec-proxy-overview), or contact your Customer Success Manager or Sauce Labs Sales Engineer.
 
 ## Choosing the Right Solution
 
-Sauce Connect Proxy and Sauce IPSec Proxy accomplish the same thing: establishing a secure connection between apps hosted on a customer's private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
+Sauce Connect Proxy and Sauce IPSec Proxy accomplish the same thing: establish a secure connection between apps hosted on a customer's private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
```
Comment (lysannep) [inline docs/secure-connections.md:102]:
```suggestion
Sauce Connect Proxy and Sauce IPSec Proxy allow you to establish a secure connection between apps hosted on a private network and Sauce Labs cloud virtual machines and real devices. The main differences are:
```

Comment (lysannep) [reply docs/secure-connections.md:102]: Although out of scope for this PR, the descriptions in the table below refer to the reader as "customer", which we should change to "you/your".

**Review by lysannep — CHANGES_REQUESTED** [review]
Added a few suggestions.

**Comment by kimsauce** [comment]
Hi @lysannep , I've reverted /secure-connections.md back to its original state and will create a separate PR for that. Can you pls approve the "overview" change?

**Review by lysannep — APPROVED** [review]

**Pull request merged**